utils/middlewares/exchange-jwt-validation-middleware.ts

```diff
@@ -0,0 +1,74 @@
+import { IRequestMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/request_middleware";
+import { getValidateJWT } from "@pagopa/ts-commons/lib/jwt_with_key_rotation";
+import {
+  IResponseErrorForbiddenNotAuthorized,
+  getResponseErrorForbiddenNotAuthorized
+} from "@pagopa/ts-commons/lib/responses";
+import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as E from "fp-ts/Either";
+import * as TE from "fp-ts/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import * as jwt from "jsonwebtoken";
+
+import { AuthBearer } from "../../generated/definitions/external/AuthBearer";
+import { IConfig } from "../config";
+import { TokenTypes } from "../enums/TokenTypes";
+
+export interface IExchangeJwtPayloadExtended extends jwt.JwtPayload {
+  readonly name: string;
+  readonly family_name: string;
+  readonly fiscal_number: FiscalCode;
+  readonly token_type: TokenTypes;
+}
+
+export type HslJWTValid = (
```

Comment:
this signature is identical to the one of the other jwt verification middleware. maybe we can create an utility function to generalize the verification.